packages/apero-core/apero-core.0.4.7/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+version:"0.4.7"
+maintainer:   ["contact@adlink-labs.tech"]
+authors:      ["Angelo Corsaro" "Julien Enoch" "Oliver Hecart" "Gabriele Baldoni"]
+homepage:     "https://github.com/atolab"
+bug-reports:  "https://github.com/atolab/apero-core/issues/"
+dev-repo:     "git+https://github.com/atolab/apero-core.git"
+
+license: "Apache-2.0"
+
+
+build: [
+ ["dune" "build" "--only" "apero-core" "--root" "." "-j" jobs "@install"]
```

Comment:
The preferred command to use during releases is: `["dune" "build" "-p" name "-j" jobs"]`